src/Components/ProvisioningWizard/steps/SourceMissing/DirectProviderLink.js

```diff
@@ -31,21 +31,21 @@ const DirectProviderLink = ({ image }) => {
         uploadStatus.options.image_name;
       break;
     case GCP_PROVIDER:
+      text = 'Launch with Google Cloud Console';
       break;
     default:
       throw new Error(`Steps requested for unknown provider: ${image.provider}`);
   }
 
   // Currently only AWS, so the ami is hardcoded
```

Comment:
I guess this comment is not relvant

Reply:
not really telling the story, that's for sure xD

Reply:
I've changed it to only the relevant part (that it's used for AWS only)